docs/reference/indices/seal.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[indices-seal]]
+== Seal
+
+The seal API allows to flush one or more indices and adds a marker to primaries and replicas if there are no pending write operations.
+The seal marker is used during recovery after a node restarts. If a replica is allocated on a node which already has a shard copy with the same seal as the primary then no files will be copied during recovery.
+Sealing is a best effort operation. If write operations are ongoing while the sealing is in progress then writing the seal might fail on some copies.
+
+[source,js]
+--------------------------------------------------
+$ curl -XPOST 'http://localhost:9200/twitter/_seal'
+--------------------------------------------------
+
+The response contains details about for which shards a seal was written and the reason in case of failure.
+Response in case all copies of a shard successfully wrote the seal:
```

Comment:
I think it'd be surprising to get a 200 response from _seal if it didn't fully seal. Folks won't check for the response if they see the 200 - they will just expect everything to have worked.

I'm not super sure what the right response should be. I suspect it depends on if you consider being unable to fully seal an index to be the server's fault ([5xx](http://tools.ietf.org/html/rfc7231#section-6.6)) or the client's ([4xx](http://tools.ietf.org/html/rfc7231#section-6.5). I _think_ its the client's fault for sending the request while some other client is sending another index request. From what I can see (conflict)[http://tools.ietf.org/html/rfc7231#section-6.5.8] might be the closest because it implies that the client might be able to resolve the issue. _seal isn't a resource in the same way that a document is but its the closest thing I see. Better than 418.